### DISCUSSION TITLE
Mouse issue when in emulation (not in AmiBerry GUI) on Chromebook with trackpad

### DISCUSSION THREAD

**Discussion by stuartmingay** [BlitterStudio/amiberry#1482] [Q&A]
Hi

I have Amiberry running on a Chromebook in its Linux env. The Chromebook is ARM64 based and has a trackpad.

The emulation works fine, but trying to move the mouse with the trackpad doesn't work. I've tried all sorts of settings, but the behaviour doesn't chage: the pointer sits top left, not at the top of the screen, and just "shakes" when I use the trackpad. There's some input being received, but it's not moving the pointer.

Has anyone else had this and does anyone know how I can solve the issue?

Thanks

Stuart

**Comment by midwan** [comment]
That sounds like the behavior if a joystick was configured on that port.
What do you have configured on Port 0, in the GUI Input panel?

**Comment by stuartmingay** [comment]
Hi
  
​
  
​Thanks for replying to my post. I have "System mouse" configured on port 0:
  
​
  
​
  
​Thanks
  
​
  
​Stuart
  
​

**Reply by midwan** [reply]
Does the trackpad show up as a different entry there, perhaps?
Hard to say, without being able to recreate this anywhere else. But it definitely seems like whatever is set as the Mouse, behaves or is interpreted as a Joystick device, instead.

**Comment by giantclambake** [comment]
This reminds me of something else ~ could it be the trackpad is using libgestures instead of libinput? AFAICT, ChromeOS by default uses the libgestures method, which works in wayland, but it has to use libinput for the trackpad to work in Xorg. There should be a setting to let libinput to apply to the trackpad in ChromeOS....I would check that.

**Comment by stuartmingay** [comment]
Thanks for replying.   
  

  
Is this a setting in the GUI or a feature flag or runtime switch I can apply?
  

  

  

  
  
  
  
>

**Reply by giantclambake** [reply]
All I could find was a reference to it in Chrome OS help...

![ex](https://imgur.com/sE51ogl.png)
